NoMansLand/Assets/Prefabs/spaceShip_002 Variant.prefab

```diff
@@ -0,0 +1,104 @@
+%YAML 1.1
```

Comment:
I am good with all the changes, just one nitpick- can we just call the spaceship prefab 
`Spaceship.prefab`? It's easier to reference in code for the future.

Reply:
Fixed ✔️